web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -326,11 +361,7 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
             csrf
           );
           const data =
-            res &&
-            res.payload &&
-            res.payload.data &&
-            res.payload.status === 200 &&
-            !res.error
+            res?.payload?.data && res?.payload?.status === 200 && !res?.error
```

Comment:
I imagine we need better error handling here like display a message in a dialog.
I'm not familiar with how other similar errors are handled in Check-ins.

Reply:
Added error handling to all the old code.